### ISSUE TITLE
stack -full panics in prettyprint with runtime error: index out of range

### ISSUE THREAD

**Issue by pquerna** [derekparker/delve#308]
Delve version: 0.10.0-alpha
go version go1.5.1 linux/amd64

`stack` worked fine, but `stack -full` caused a panic. 

```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/derekparker/delve/service/api.(*Variable).writeTo(0xc822009000, 0xc821763110, 0x10000, 0xc821e7de70, 0x2)
	/home/chip/src/github.com/derekparker/delve/service/api/prettyprint.go:64 +0xb9f
github.com/derekparker/delve/service/api.(*Variable).writeStructTo(0xc821fd3d40, 0xc821763110, 0x100, 0xa52810, 0x1)
	/home/chip/src/github.com/derekparker/delve/service/api/prettyprint.go:128 +0x8b9
github.com/derekparker/delve/service/api.(*Variable).writeTo(0xc821fd3d40, 0xc821763110, 0x10000, 0xa52810, 0x1)
	/home/chip/src/github.com/derekparker/delve/service/api/prettyprint.go:67 +0x136f
```

full details in gist:
https://gist.github.com/pquerna/bec1376500c056315446

**Comment by aarzilli** [comment]
Probably fixed by ef89e7e6ac5560b02805de3a8096ee8f80704513

**Comment by aarzilli** [comment]
This should be fixed by 2deb7fba2021437c0831dae7736a46efd0aecb03, please reopen if this is not the case.
